### PR TITLE
TZIP-XXX pattern is not recognized

### PR DESCRIPTION
**Pull request by ungarson**

### CODE REVIEW

**Comment by ungarson** [comment]
[TZIP-XXX pattern is not recognized](https://trello.com/c/jJhTy6na/985-tzip-xxx-pattern-is-not-recognized)

File: src/views/contract/MetadataTab/CustomFields/InterfacesField.vue [hunk 1]
```diff
@@ -36,11 +36,11 @@ export default {
   },
   methods: {
     getInterfaceLink(interfaceName) {
-      const constant = `https://gitlab.com/tzip/tzip/-/blob/master/proposals`;
-      return `${constant}/${interfaceName.toLowerCase()}`
+      const constant = `https://gitlab.com/tzip/tzip/-/tree/master/proposals`;
+      return `${constant}/${interfaceName.toLowerCase().split('-0').join('-')}`
```
Comment (m-kus) [inline src/views/contract/MetadataTab/CustomFields/InterfacesField.vue:40]: `"TZIP-012 git 6544de32"` counter example

Comment (m-kus) [reply src/views/contract/MetadataTab/CustomFields/InterfacesField.vue:40]: Use regexp

Comment (ungarson) [reply src/views/contract/MetadataTab/CustomFields/InterfacesField.vue]: @m-kus thanks, fixed

**Pull request merged**